blockly/msg/json/nl.json

```diff
@@ -7,7 +7,8 @@
 			"MedShot",
 			"아라",
 			"JaapDeKleine",
-			"Sjoerddebruin"
+			"Sjoerddebruin",
+			"Benny Malengier"
```

Comment:
Only if you start using translatewiki for ardublockly translation. Otherwise it will just be a merge conflict on the next new translater when you merge new blockly (due to the , not present after my name). Once that merge conflict is corrected, future merges will go without problems I believe